reconstruction.py

```diff
@@ -626,10 +629,17 @@ def reconstruct(self,evrange=(-1,-1,-1)):
                                 indx = trg * nChannels_f + ch
                                 waveform_info = { 'run' : run, 'event': event, 'channel' : ch, 'trigger' : trg , 'GE' : insideGE, 'sampling' : "fast", 'TTT' : (TTTs_f[trg]*8.5/1000./1000.)}
 
+                                t0 = time.perf_counter()
```

Comment:
It could be a good idea to use different variable names for the t0 and t1.
And we could also think about cleaning the variable after each to be sure to not save old or memory values by mistake.